resources/js/processes-catalogue/components/ProcessTab.vue

```diff
@@ -5,29 +5,359 @@
+  components: {
+    PMColumnFilterPopover,
+    paginationTable,
+    TasksList,
+    DefaultTab,
+  },
+  mixins: [uniqIdsMixin, ListMixin],
+  props: {
+    currentUser: {
+      type: Object,
+    },
+    process: {
+      type: Object,
+    },
+  },
+  data() {
+    return {
+      pmqlRequest: `(status = "In Progress") AND (requester = "${this.currentUser.username}")`,
+      pmqlTask: `(status = "In Progress")`,
+      filter: "",
+      previousFilter: "",
+      previousPmql: "",
+      orderBy: "id",
+      orderDirection: "DESC",
+      additionalParams: "",
+      advanced_filter: "",
+      sortOrder: [
+        {
+          field: "id",
+          sortField: "id",
+          direction: "desc",
+        },
+      ],
+      fields: [],
+      showTabRequests: false,
+      showTabTasks: false,
+      tableHeaders: [],
+      tableHeadersRequests: [
+        {
+          label: "REQUEST",
+          field: "name",
+          sortable: true,
+          default: true,
+          width: 110,
+          truncate: true,
+        },
+        {
+          label: "STATUS",
+          field: "status",
+          sortable: true,
+          default: true,
+          width: 150,
+        },
+        {
+          label: "DUE DATE",
+          field: "due_at",
+          format: "datetime",
+          sortable: true,
+          default: true,
+          width: 120,
+        },
+      ],
+      tableHeadersTasks: [
+        {
+          label: "NAME",
+          field: "element_name",
+          sortable: true,
+          default: true,
+          width: 140,
+          truncate: true,
+        },
+        {
+          label: "REQUEST",
+          field: "request",
+          sortable: true,
+          default: true,
+          width: 140,
+          truncate: true,
+        },
+        {
+          label: "STATUS",
+          field: "status",
+          sortable: true,
+          default: true,
+          width: 100,
+        },
+        {
+          label: "DUE DATE",
+          field: "due_at",
+          format: "datetime",
+          sortable: true,
+          default: true,
+          width: 140,
+        },
+      ],
+      dataRequests: {},
+      dataTasks: {},
+      savedSearch: false,
+      queryTask: "",
+      queryRequest: "",
+      perPage: 10,
+    };
+  },
+  computed: {
+    endpoint() {
+      if (this.savedSearch !== false) {
+        return `saved-searches/${this.savedSearch}/results`;
+      }
+
+      return "requests";
+    },
+  },
+  mounted() {
+    this.queryBuilder();
+  },
+  methods: {
+    jsonRows(rows) {
+      if (rows.length === 0 || !_.has(_.head(rows), "_json")) {
+        return rows;
+      }
+      return rows.map((row) => JSON.parse(row._json));
+    },
+    changePage(page) {
+      this.page = page;
+      this.queryBuilder();
+    },
+    handleRowClick(type, row) {
+      const openFunction = type === 'request' ? this.openRequest : this.openTask;
+      window.location.href = openFunction(row, 1);
```

Comment:
where is defined this method

Reply:
is defined in "filter-table" component calling this way:
@table-row-click="(row) => handleRowClick('request', row)"
This methods selects a row and opens a URL